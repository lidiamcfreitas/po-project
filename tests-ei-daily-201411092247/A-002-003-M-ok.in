3
aaa
1
6
2
0
0
