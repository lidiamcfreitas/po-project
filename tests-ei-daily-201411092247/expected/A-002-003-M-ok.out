Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: root:super user:/home/root
Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: 